contracts/token/MultiCurrency.sol

```diff
@@ -8,91 +8,68 @@ pragma solidity ^0.8.0;
 /// @dev This contracts will interact with currencies pallet
 library MultiCurrency {
     /// @dev The MultiCurrency precompile address.
-    address private constant PRECOMPILE =
-        address(0x0000000000000000000000000000000000000400);
+    address private constant PRECOMPILE = address(0x0000000000000000000000000000000000000400);
 
     function name() internal view returns (string memory) {
-        (bool success, bytes memory returnData) = PRECOMPILE.staticcall(
-            abi.encodeWithSignature("name()")
-        );
+        (bool success, bytes memory returnData) = PRECOMPILE.staticcall(abi.encodeWithSignature("name()"));
         assembly {
-            if eq(success, 0) {
-                revert(add(returnData, 0x20), returndatasize())
-            }
+            if eq(success, 0) { revert(add(returnData, 0x20), returndatasize()) }
```

Comment:
I am not sure if I like this formatter. What tool are you using for format? At least the max line width should be reduced.
Best to avoid reformat code and modify it in a same PR because it can be hard to review the actual change